test/common/upstream/hds_test.cc

```diff
@@ -215,8 +217,8 @@ TEST_F(HdsTest, TestProcessMessageHealthChecks) {
   hds_delegate_friend_.processPrivateMessage(*hds_delegate_, std::move(message));
 
   // Check Correctness
-  EXPECT_EQ(hds_delegate_->hdsClusters()[0]->healthCheckers().size(), 2);
-  EXPECT_EQ(hds_delegate_->hdsClusters()[1]->healthCheckers().size(), 3);
+  EXPECT_EQ(hds_delegate_->hdsClusters()["minkowski0"]->healthCheckers().size(), 2);
+  EXPECT_EQ(hds_delegate_->hdsClusters()["minkowski1"]->healthCheckers().size(), 3);
```

Comment:
Did you have any further thoughts on dedicated unit tests for the in-place updates?

Reply:
Yeah, I'll probably have more time next week to finish them.

Reply:
Great, thanks.